editor/components/inserter/style.scss

```diff
@@ -31,8 +35,61 @@
 	border: 1px solid $light-gray-500;
 	position: absolute;
 	left: -122px;
-	bottom: 42px;
 	background: #fff;
+
+	&.is-top {
+		bottom: 42px;
+
+		.editor-inserter__arrow {
+			bottom: -10px;
+			left: 50%;
+			margin-left: -10px;
+			border-top-style: solid;
+			border-bottom: none;
+			border-left-color: transparent;
+			border-right-color: transparent;
+
+			&:before {
+				bottom: 2px;
+				border: 10px solid $white;
+				content: " ";
+				position: absolute;
+				left: 50%;
+				margin-left: -10px;
+				border-top-style: solid;
+				border-bottom: none;
+				border-left-color: transparent;
+				border-right-color: transparent;
+			}
+		}
+	}
+
+	&.is-bottom {
+		top: 42px;
+
+		.editor-inserter__arrow {
+			top: -10px;
+			left: 50%;
```

Comment:
Seems a decent amount of these styles could have been left in the common unscoped `.editor-inserter__arrow` selector.

`left`
`margin-left`
`border-left-color`
`border-right-color`
`&:before content`
`&:before position`
`&:before position`
`&:before left`
`&:before border-left-color`
`&:before border-right-color`

I guess a point could be made to having related styles colocated despite repetition.